httputil/retry.go

```diff
@@ -139,6 +147,20 @@ func RetryRequest(endpoint string, doRequest func() (*http.Response, error), rea
 			if ShouldRetryError(attempt, err) {
 				continue
 			}
+
+			// on 16.04 the error below is "temporary", however on 18.04 it is not, so we can't relay on Temporary() flag and instead need
```

Comment:
Nitpick: please try to capitalise sentences.

```suggestion
			// On 16.04 the error below is "temporary", however on 18.04 it is not, so we can't relay on Temporary() flag and instead need
```